pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/Namespaces.java

```diff
@@ -843,5 +843,30 @@ public void setSchemaAutoUpdateCompatibilityStrategy(@PathParam("tenant") String
         internalSetSchemaAutoUpdateCompatibilityStrategy(strategy);
     }
 
+    @GET
+    @Path("/{tenant}/{namespace}/schemaValidationEnforced")
+    @ApiOperation(value = "")
```

Comment:
Can you please write a documentation for this operation?

There is a description of what the setting means. You can use that as a material.
https://github.com/apache/pulsar/blob/master/conf/broker.conf#L634